CHANGELOG.next.asciidoc

```diff
@@ -216,6 +216,7 @@ Setting environmental variable ELASTIC_NETINFO:false in Elastic Agent pod will d
 - Parse more fields from Elasticsearch slowlogs {pull}38295[38295]
 - Update CEL mito extensions to v1.10.0 to add keys/values helper. {pull}38504[38504]
 - Add support for Active Directory an entity analytics provider. {pull}37919[37919]
+- Add AWS AWSHealth metricset. {pull}38370[38370]
```

Comment:
```suggestion
- Add AWS's AWSHealth metricset. {pull}38370[38370]
```

Reply:
This is following the existing changelog conversion - [Main Module]AWS [Sub Module name] convension